pkgs/applications/networking/cluster/terraform-providers/providers.txt

```diff
@@ -7,7 +7,7 @@
 #  <organisation>/<repo>                                         - include only the named repository.
 
 # include all terraform-providers
-terraform-providers terraform-provider- terraform-provider-\\(azure-classic\\|scaffolding\\|google-beta\\|skytap\\)
+terraform-providers terraform-provider- terraform-provider-\\(azure-classic\\|scaffolding\\)
```

Comment:
Do you know why those where disabled in the first place?

Reply:
from what i can tell, these providers weren't tagged with git versions for a long time (which prevented the `update-all` script from working).

I can remove these if needed, just saw that there were versioned now

Reply:
google-beta is also desirable to have, not sure about skytap

they are pulling out all the beta features from the google provider and moving them into google-beta, but a lot of google feature stay in beta for a long time